packages/manager/modules/vps/src/upscale/configuration-price/configuration-price.component.js

```diff
@@ -0,0 +1,24 @@
+import controller from './configuration-price.controller';
+import template from './configuration-price.html';
+
+/**
+ * Examples:
+ * <vps-upgrade-configuration-price price="{ value: 32.30, unit: 'microcents', currency: 'EUR' }" language="'fr_FR'"></vps-upgrade-configuration-price>
+ * <vps-upgrade-configuration-price price="{ value: 32.30, unit: 'cents', currency: 'USD' }" language="'fr_CA'"></vps-upgrade-configuration-price>
+ * <vps-upgrade-configuration-price price="{ value: 32.30, unit: 'cents', currency: 'EUR' }" language="'en_US'"></vps-upgrade-configuration-price>
+ */
+export default {
+  bindings: {
+    /**
+     * string
+     * formatted the OVHcloud way
+     */
+    language: '<',
```

Comment:
Why do you have to pass the language ? Why not get it in the component ?

Reply:
Maybe it is because the lanaguage correspond to the user language, not the`universe_selected_language` ?

Reply:
I'm having trouble understanding. Do you mean user language selected for other apps that is equal to `me.language` from `/me` ?

Reply:
Exactly ! :)

Reply:
I wanted it to be as stupid as stupid does, this way it the less dependencies possible (no constructor is needed), but I can include it if it makes more sense to you